docs/components/bucket.md

```diff
@@ -0,0 +1,168 @@
+# Bucket
+
+The bucket component of Thanos is a set of commands to inspect data in object storage buckets.
+It is normally run as a stand alone command to aid with troubleshooting. 
+
+Example:
+
+```
+$ thanos bucket verify -r --gcs.bucket example-bucket
```

Comment:
please let's not show `-r` in the example (: It is not safe. (you need to ensure global compactor is not running etc)